File: qe-selenium-python/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/keyboard-events")
    print(driver.title)
    actions = webdriver.ActionChains(driver)
    actions.send_keys("Hello World!").perform()
    text = driver.find_element(By.CSS_SELECTOR, 'h1.mt-3').text
    print(text)

File: qe-selenium-java/Activity9.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity9 {
    public static void main(String[] args) {
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/keyboard-events");
        System.out.println("Title of the Page: "+driver.getTitle());

        Actions actions = new Actions(driver);

        actions.sendKeys("Hello World!").perform();

        String text = driver.findElement(By.cssSelector("h1.mt-3")).getText();
        System.out.println(text);

        
        driver.quit();

    }
}
